### ISSUE TITLE
Doesn't click

### ISSUE THREAD

**Issue by 05v** [Blakley/Clash#1]
Tried the script, and it doesn't seem to actually press down on anything :(

**Comment by NORXONDOR** [comment]
Hi. A couple things I recommend checking:

1. Make sure after inputting 'Number of players to invite:' that you click into your BlueStacks window to focus it.

2. Make sure you're running BlueStacks in full-screen mode (CoC should fill your whole monitor), with BlueStacks 'Display resolution' at 1920x1080. 

3. Make sure the monitor you're running BlueStacks on is set to 1920x1080 resolution. Otherwise, you'll need to customise the "self.positions" and "self.player_positions" coordinate tuples to suit your resolution.

If you run it again and notice UI elements/buildings being selected, this issue has been fixed.

**Comment by Blakley** [comment]
Yeah sorry about that! It's been a while since I looked at this project. Thanks a ton Norx for the response and fix.
